build/admission/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.16-alpine3.13 AS builder
+FROM golang:1.17.13-alpine3.15 AS builder
```

Comment:
Is there an official list to determine the corresponding image `golang:1.17.13-alpine3.15`?

Reply:
golang:1.17.13-alpine3.15 This image does not work, there is a critical vulnerability in zlib, the latest image is also golang:1.17.13-alpine3.16, only updating to alpine3.17.1 can ensure that the base image has no loopholes, otherwise only Can be upgraded to solve @vincentgoat @gy95

Reply:
you mean use golang:1.17.13-alpine3.16, but not golang:1.17.13-alpine3.15?

Reply:
golang:1.17.13-alpine3.15
<img width="1147" alt="image" src="https://user-images.githubusercontent.com/18587688/211965763-61ab9524-d26b-4c77-b1c4-2961a3825246.png">

golang:1.17.13-alpine3.16
<img width="1056" alt="image" src="https://user-images.githubusercontent.com/18587688/211965788-4b0eeae9-0398-4c6c-9377-741a4c96f83b.png">

I mean to use the version of alpine3.17.1, but golang:1.17.13 does not have the version of alpine3.17, so it can only be solved by upgrading to

Reply:
Based on the image golang:1.17.13-alpine3.16, is there a way to fix the vulnerability by upgrading zlib version? @fengshunli

Reply:
yes

Reply:
Sounds good 👍  let’s use `1.17.13-alpine3.16` here @gy95 Then fix the cve in this base image

Reply:
Then I submit a pr to solve the problem of zlib @fisherxu

Reply:
see https://github.com/kubeedge/kubeedge/pull/4542 @gy95 @fisherxu @vincentgoat

Reply:
fixed

Reply:
You can remove your changes to the dockerfile, or my pr will be merged, and yours will conflict @gy95

Reply:
we better use alpine3.16, keep the same with alpine version of image golang:1.17.13-alpine3.16

Reply:
the new version is the official version that has been fixed. If you use the old version, there will be loopholes later, and you need to upgrade. Why not keep it consistent with the official image?

Reply:
what's the difference between alpine3.16 and alpine3.17?

Reply:
> we better use alpine3.16, keep the same with alpine version of image golang:1.17.13-alpine3.16

Agree. We can fix the vulnerability by upgrading zlib version, based on the image golang:1.17.13-alpine3.16.

Reply:
updated